bcda/models/models_test.go

```diff
@@ -585,12 +590,49 @@ func (s *ModelsTestSuite) TestGetEnqueJobs_Coverage() {
 		enqueuedBenes += len(jobArgs.BeneficiaryIDs)
 		assert.True(len(jobArgs.BeneficiaryIDs) <= 5)
 		assert.Equal(since, jobArgs.Since)
+		assert.Equal(20, jobArgs.Priority)
 		assert.NotNil(jobArgs.TransactionTime)
 	}
 	assert.Equal(50, enqueuedBenes)
 	os.Unsetenv("BCDA_FHIR_MAX_RECORDS_COVERAGE")
 }
 
+func (s *ModelsTestSuite) TestGetEnqueJobs_WithSyntheticACOs() {
+	assert := s.Assert()
+
+	j := Job{
+		// ACOID:      uuid.Parse("ACO9991"),
+		ACOID:      uuid.Parse(constants.DevACOUUID),
+		RequestURL: "/api/v1/Patient/$export?_type=Patient",
+		Status:     "Pending",
+	}
+	s.db.Save(&j)
+	defer s.db.Delete(&j)
+
+	since := ""
+	err := os.Setenv("SYNTHETIC_ACO_IDS", constants.DevACOUUID)
```

Comment:
Can this be set in `docker-compose` as a comma-delimited list, so it represents exactly what will be defined in AWS?
Can look something like this in `docker-compose`:
```
A9990, A9991, A9992, A9993, A9994
```